css-brochure-form/styles.css

```diff
@@ -51,7 +51,7 @@ button {
   border-width: 0;
   border-radius: 2em;
   padding: 0.7em 1.4em;
-  font-family: 'AtlasGrotesk';
+  font-family: 'AtlasGrotesk', sans-serif;
   font-style: normal;
   font-weight: 300;
   src: url('AtlasGrotesk-Light-Web.woff2') format('woff2');
```

Comment:
These lines are only necessary in the `@font-face` rule, you can remove them here.